core/main/src/processor/metrics_processor.rs

```diff
@@ -199,9 +199,10 @@ impl ExtnRequestProcessor for MetricsProcessor {
                     Err(e) => Self::handle_error(client, msg, e).await,
                 }
             }
-            MetricsPayload::OperationalMetric(t) => {
+            MetricsPayload::TelemetryPayload(t) => {
```

Comment:
@bobra200: This is a change you made that I pulled in from your POC. Could you explain the reasoning behind the rename? I'm still a bit unclear between the differences between what we're calling "operational metrics" vs "behavioral metrics" vs "telemetry".

Reply:
Yeah, it's ... murky for sure: 

"Behavioral Metrrics" are high  cardinality metrics that describe user interactions
"Operational Metrics" are low cardinality metrics that are meant to describe how the "fleet" of devices is behaving at an aggregate  - for instance, how long does a given FB api take for a given OS version - theses metrics are aggregated as histograms
"Telemetry" - log based , high cardinality meant ostensibly for triage.